index.html

```diff
@@ -746,6 +746,148 @@ <h3>JcsEd25519Key2020</h3>
     </pre>
   </section>
 
+</section>
+
+<section>
+  <h2>BBS+ Signature 2020</h2>
+  
+  <section>
+
+  <h3>BbsBlsSignature2020</h3>
+
+  <table class="simple">
+    <thead>
+      <tr>
+        <th colspan=2>Summary</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr>
+        <td>Identifiers</td>
+        <td>BbsBlsSignature2020</td>
+      </tr>
+      <tr>
+        <td>Status</td>
+        <td>PROVISIONAL</td>
+      </tr>
+      <tr>
+        <td>Authors</td>
+        <td>Tobias Looker, Orie Steele</td>
+      </tr>
+      <tr>
+        <td>Specification</td>
+        <td><a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Signatures 2020</a></td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h3>BbsBlsSignatureProof2020</h3>
+
+  <table class="simple">
+    <thead>
+      <tr>
+        <th colspan=2>Summary</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr>
+        <td>Identifiers</td>
+        <td>BbsBlsSignatureProof2020</td>
+      </tr>
+      <tr>
+        <td>Status</td>
+        <td>PROVISIONAL</td>
+      </tr>
+      <tr>
+        <td>Authors</td>
+        <td>Tobias Looker, Orie Steele</td>
+      </tr>
+      <tr>
+        <td>Specification</td>
+        <td><a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Signatures 2020</a></td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h3>Bls12381G1Key2020</h3>
+
+  <table class="simple">
+    <thead>
+      <tr>
+        <th colspan=2>Summary</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr>
+        <td>Identifiers</td>
+        <td>Bls12381G1Key2020</td>
```

Comment:
Same comment as above.

Also, really difficult to understand what 12381G1 means... I should note that I committed the original sin of `Ed25519VerificationKey2020`... We need to have a talk w/ the cryptographers and tell them to stop naming things so horribly. This is an example of implementation details leaking to people that shouldn't know about those implementation details.

Reply:
Agree really trying not expose any additional complexity here and its a tough tradeoff to make. 12-381 is the curve definition and on a BLS curve because it is paring friendly you can have a public key in either or both of the G1 and G2 fields. 

Why do we need both why not pick one? Well in the simple cases you only need G2 as this is the key needed by an issuer to be able to create a BBS signature. However, if the subject of the signature wants a public key bound into the signature in a way that it is never revealed as a correlating identifier when deriving proofs, then they need this to be in the opposite field to the issuers key (G1). 

Essentially the general use case this enables is for a subject of a verifiable credential to have a G1 public key in their did document that can be used by an issuer with a G2 public key to construct a bound BBS signature.

Reply:
The string name is not meant to explain the key type exhastively... thats what the spec text is for... this is a registry... this is not the place to define all the terms used by a suite... its a place to register suites.